src/rules/component-store/updater-explicit-return-type.ts

```diff
@@ -1,5 +1,6 @@
+import type { TSESTree } from '@typescript-eslint/experimental-utils'
+import { ESLintUtils } from '@typescript-eslint/experimental-utils'
 import path from 'path'
-import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils'
 
 import { docsUrl, findNgRxComponentStoreName } from '../../utils'
```

Comment:
Question: is this blank line between the 3rd. imports and the project imports on purpose? I ask because I saw it is present in some files and not others... To be honest, I think not having this line is much better since I can just save and the VSCode (in my case) organizes everything automatically.

Reply:
I don't have a strong opinion on this, if vscode can sort it automatically we should just respect that and remove the line. In some projects this is a convention to differentiate libs with imports from your own code base. So if you want, you can remove the extra line before this gets merged.

Reply:
Done! To make this change in all files at once, I've installed the extension [folder-source-actions](https://marketplace.visualstudio.com/items?itemName=bierner.folder-source-actions), but normally(

```json
    "editor.codeActionsOnSave": {
      "source.organizeImports": true
    }
```

) on `settings.json` is sufficient for me 😄

Reply:
Good to know that there's a plugin for it!
I think we could also enforce this with the [sort-imports rule](https://eslint.org/docs/rules/sort-imports)?

Reply:
Hmmmmm, this rule acts a little different... it doesn't just allow us to sort the imports alphabetically, no matter what type, or rather `memberSyntaxSortOrder`, like the VSCode command.

What I see people doing, if they want to leave it in lint too, is to use a 3rd. plugin like [**`eslint-plugin-import/order.md`**](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) or a simpler version like [**`simple-import-sort/imports`**](https://github.com/lydell/eslint-plugin-simple-import-sort/issues/65)... however, these still are not compatible with simply _organize my imports_, as it still requires us to choose a `groups` configuration (or to use the default one).

Looking for way to truly behave like the VSCode `organizeImports`, I found this [issue](https://github.com/lydell/eslint-plugin-simple-import-sort/issues/65) and found two interesting approaches to go with [here](https://github.com/lydell/eslint-plugin-simple-import-sort/issues/65#issuecomment-763006464): either enforce this using prettier through [**`prettier-plugin-organize-imports`**](https://github.com/simonhaenisch/prettier-plugin-organize-imports) or [**`organize-imports-cli`**](https://github.com/thorn0/organize-imports-cli). The first option (`prettier-plugin-organize-imports`) seems to be more popular, but it's up to you whether to add this here (IMHO it's a nice thing to have).

Reply:
Feel free to add it if you want @rafaelss95 👍

Reply:
Done!